core/trino-main/pom.xml

```diff
@@ -212,6 +212,11 @@
             <artifactId>jsr305</artifactId>
```

Comment:
Don't you need to add `@FormatString` to airlift logger methods?

Reply:
There's already one, and that's how we can find all these issues here:

```java
    @FormatMethod
    public void debug(String format, Object... args)
    {
        ...
    }
```